### DISCUSSION TITLE
Join starlite, instead?

### DISCUSSION THREAD

**Discussion by dkress59** [adriangb/xpresso#23] [Q&A] [answered]
Hey there, I just came across this repo and couldn't help but notice that it's awfully similar to the [Starlite framework](https://github.com/starlite-api/starlite), which I recently got involved with.

It looks like you are going for the same kind of framework, so I suggest you get involved with the young but lively Starlite community! The very helpful people there are driven to design the best possible framework, you really don't need to do all of the work all by yourself.

Take care ✌️

**Comment by adriangb** [comment, marked as answer]
I don't think this is any more similar to Starlite than Starlite is to FastAPI. There's plenty of fundamentally different design decisions.

**Reply by dkress59** [reply]
I see you have forked starlite and the related repos, but you don't mention it in your docs, anywhere, so I am happy to hear that.

See you around 🤙

**Reply by dkress59** [reply]
thanks for downvoting. I'm not sure I could have been any more diplomatic.

I guess that's just the kind of person you are? @Kludex

**Comment by Kludex** [comment]
The way I see it, the core ideas of those two frameworks are fundamentally different.

What are the similarities you see?

**Reply by dkress59** [reply]
Read the title and read my suggestion. @adriangb already gave me a sufficient answer.

**Comment by adriangb** [comment]
This does not seem like it will lead to a productive discussion. Sorry @dkress59 but I'm going to lock this discussion. Best of luck with your development work.
